CSLisp/src/core/Primitives.cs

```diff
@@ -200,12 +197,47 @@ public class Primitives
                 var member = TypeUtils.GetMemberFieldOrProp(type, memberName);
                 TypeUtils.SetValue(member, instance, targetValue.AsBoxedValue);
                 return targetValue;
-
             }), sideFx: SideFx.Possible),
-        };
 
+            new Primitive("make-vector", 1, new Function((Context ctx, Val a) => {
+                if(a.IsNumber) {
+                    return new Val( new List<Val>(Enumerable.Repeat((Val)0, a.AsInt)));
+                }
```

Comment:
Can you please have your IDE apply formatting rules in .editorconfig ?
This is so that we keep consistency in spacing etc.
In Visual Studio, this can be done by running "Run Code Cleanup", or CTRL-K, CTRL-E

Reply:
It does not work

Reply:
Hmm, that's ok, I'll run the formatter next time around